tyrian/shared/src/main/scala/tyrian/Attr.scala

```diff
@@ -86,5 +86,29 @@ object Property:
   * @param msg
   *   Message to produce when the event is triggered
   */
-final case class Event[E <: Tyrian.Event, M](name: String, msg: E => M) extends Attr[M]:
-  def map[N](f: M => N): Attr[N] = Event(name, msg andThen f)
+final case class Event[E <: Tyrian.Event, M](
+    name: String,
+    msg: E => M,
+    preventDefault: Boolean,
+    stopPropagation: Boolean,
+    stopImmediatePropagation: Boolean
+) extends Attr[M]:
+  def map[N](f: M => N): Attr[N]                  = Event(name, msg andThen f)
+  def preventDefaultToggle: Event[E, M]           = this.copy(preventDefault = !preventDefault)
```

Comment:
I can't think of a use case for this, did you have something in mind?

Reply:
Having stewed on it a bit I think you're right. I can't think of a use-case either. :smile: :+1: